### PR TITLE
Try to load flink-conf.yaml from FLINK_CONF_DIR first

### PR DESCRIPTION
**Pull request by grzegorz8**

### CODE REVIEW

File: streaming_jupyter_integrations/magics.py [hunk 1]
```diff
@@ -623,17 +625,26 @@ def __retract_user_as_something_is_executing_in_background() -> None:
 
     @staticmethod
     def __read_global_config() -> Dict[str, Any]:
```
Comment (swtwsk) [inline streaming_jupyter_integrations/magics.py:627]: Test it please 😿

**Review by swtwsk — CHANGES_REQUESTED** [review]
LGTM, but add few short tests for searching for the env vars and the file, please

File: streaming_jupyter_integrations/magics.py [hunk 2]
```diff
@@ -623,17 +625,26 @@ def __retract_user_as_something_is_executing_in_background() -> None:
...
-                    print(exc)
+        if "FLINK_CONF_DIR" in os.environ:
+            config_path = os.path.join(os.environ["FLINK_CONF_DIR"], "flink-conf.yaml")
+            return Integrations.__read_flink_conf_yaml(config_path)
+        elif "FLINK_HOME" in os.environ:
+            config_path = os.path.join(os.environ["FLINK_HOME"], "conf", "flink-conf.yaml")
+            return Integrations.__read_flink_conf_yaml(config_path)
         else:
-            print("FLINK_HOME environment variable is not set, reading flink-conf skipped")
+            print("Neither FLINK_CONF_DIR nor FLINK_HOME environment variable is set, reading flink-conf skipped.")
```
Comment (andrzejdackiewicz) [inline streaming_jupyter_integrations/magics.py]: ... environment variable was set. Skipping reading from flink-conf.

File: streaming_jupyter_integrations/magics.py (hunk 1 above)
Comment (grzegorz8) [reply streaming_jupyter_integrations/magics.py]: Tests added

**Review by swtwsk — APPROVED** [review]
🚀

**Review by andrzejdackiewicz — APPROVED** [review]

File: streaming_jupyter_integrations/magics.py (hunk 2 above)
Comment (grzegorz8) [reply streaming_jupyter_integrations/magics.py]: I think present simple is fine here.
Splitting into two sentences looks good.

**Review by niladrem — APPROVED** [review]

**Pull request merged**